### PR TITLE
fix #33082 snapdeal.com

### LINKED ISSUE #33082: snapdeal.com
### Issue URL (Ads)
[https://www.snapdeal.com/products/women-apparel](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.snapdeal.com%2Fproducts%2Fwomen-apparel)
### Comment
Username: @danidoedel
> Tracking
> 
> https://imp.snapdeal.biz/pub/imp?version=v6&exp=MLF,SSF,SCF&impData=101%7C155631212410443003%7C%7C%7Cwomen-apparel%7Cclp%7C10.70.96.19%3Aa652cb44-8852-4115-a73f-e16c38fffd25%7Cproduct%7Csimilar_ad%7C1556312278577%7C%7Bsd-cr-198X232-1-3%7D1%7E448882068%7E923966%7E100331%7E629488747257%7ESDL219894973%7ESd3523%7EA1+Fashion%7E%7ECPC%7E%7E97.24055188962207__2%7E448882066%7E923966%7E100331%7E625659365044%7ESDL074524442%7ESd3523%7EA1+Fashion%7E%7ECPC%7E%7E97.24055188962207__3%7E394267726%7E907954%7E38285%7E619788049482%7ESDL024900462%7ESd796e%7EVOGUE+NATION%7E%7ECPC%7E%7E0.0__4%7E448882069%7E923966%7E100331%7E641699635129%7ESDL830346130%7ESd3523%7EA1+Fashion%7E%7ECPC%7E%7E97.24055188962207

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | Win
AdGuard version: | 7.0.2541.6357
Browser: | Chrome
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, Send Do-Not-Track header, Remove X-Client-Data header from HTTP requests, Self-destructing third-party cookies (180), Disable cache for third-party requests, Block WebRTC, Block Location API
Filters: | AdGuard Annoyances,<br>AdGuard Base,<br>AdGuard Tracking Protection,<br>AdGuard Social Media,<br>AdGuard German

### PR DESCRIPTION
**Pull request by zloyden**
https://github.com/AdguardTeam/AdguardFilters/issues/33082

### CODE REVIEW

File: SpywareFilter/sections/tracking_servers_firstparty.txt [hunk 1]
```diff
@@ -22,6 +22,7 @@
 ||tags.news.com.au^
 !
 ||pixelappcollector.thesun.co.uk^
+||sga.snapdeal.biz^
```
Comment (BlazDT) [inline SpywareFilter/sections/tracking_servers_firstparty.txt]: Unsure about this url, when opening one you will get e. g. dresses and this is a shop.

https://sga.snapdeal.biz/pub/getContent?pubId=1&rs=html&impressionId=hello&slotId=sd-cr-198X232-1-3&siteId=101&js=true&response_type=similar_ad&min_val=4&max_val=4&inv=site&category=women-apparel&templateId=listing6&pagetype=clp&sdId=156458930472140737&q=&pincode=&eId=&zone=NO_ZONE&phn=null

Comment (zloyden) [reply SpywareFilter/sections/tracking_servers_firstparty.txt]: Request from issue generated by `sga.snapdeal.biz`. What about this rule? `||imp.snapdeal.biz/pub/imp?`

Comment (BlazDT) [reply SpywareFilter/sections/tracking_servers_firstparty.txt]: That request is not made when I visit the site.

Comment (zloyden) [reply SpywareFilter/sections/tracking_servers_firstparty.txt]: <details><summary>Screenshot:</summary>



</details><br/>

Comment (Alex-302) [reply SpywareFilter/sections/tracking_servers_firstparty.txt]: `||imp.snapdeal.biz^` is ok

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**